grammars/xml.cson

```diff
@@ -343,3 +375,94 @@
         'name': 'punctuation.definition.comment.xml'
     'end': '--%?>'
     'name': 'comment.block.xml'
+  'embedded-css':
+    'patterns': [
+      {
+        'begin': """(?ix)
+          <style
+          (?=[>:\\s])
+          (?!
+            [^>]* \\s type
+            \\s* = \\s*
+            ['"]?
+            (?!\\btext/css\\b)
+            [^'"]+
+            ['"]?
+          )
+          [^>]*
+          >
+        """
+        'beginCaptures':
+          '0':
+            'patterns': [
+              'include': '#generic-xml'
+            ]
+        'contentName': 'source.css.embedded.xml'
+        'end': '(?ix:</style>)'
```

Comment:
What do you mean? We *are* [matching case-insensitively](https://github.com/Alhadis/language-xml/blob/ff0b98fd5d90baaa07b0fad3ce472d039feca8fa/grammars/xml.cson#L381-L401).